lab_6_pipeline/pipeline.py

```diff
@@ -139,6 +226,8 @@ def analyze(self, texts: list[str]) -> list[StanzaDocument]:
         Returns:
             list[StanzaDocument]: List of documents
         """
+        analyzed = self._analyzer.process([Document([], text=' '.join(texts))])
```

Comment:
Again, why split into sentences in the first place if you then glue them back together

Reply:
fixed